src/main/java/Menu.java

```diff
@@ -60,6 +62,23 @@ public static int showPrincipalMenuAndRetrieveOption(Scanner sc) {
         }
         return selectedOption;
     }
+    private static void createTeachers() {
```

Comment:
We could extract here variables definition and just reassign in the loop
String name;
double salary;